docs/documentation/development/bindings/xml-reference.md

```diff
@@ -446,6 +448,7 @@ Bridge and *Thing* descriptions must be placed as XML file(s) (with the ending `
   <tr><td>bridge-type-ref.id</td><td>The identifier of a bridge this bridge/<i>Thing</i> can connect to (mandatory).</td></tr>
   <tr><td>label</td><td>A human-readable label for the bridge/<i>Thing</i> (mandatory).</td></tr>
   <tr><td>description</td><td>A human-readable description for the bridge/<i>Thing</i> (optional).</td></tr>
+  <tr><td>category</td><td>Category this bridge/<i>Thing</i> belongs to to be used for filtering e.g. <code>Light</code> (optional).</td></tr>
```

Comment:
remove the example and the "filtering" use - maybe rather link to the categories from here.